### PR TITLE
Fixed submit button

### PR DESCRIPTION
**Pull request by stefaniacardenas**
* Fix issue with submit verification button where text was appearing on two lines
* Elements that use the `btn` class are now consistently using the `button` tag
* Fixed position of Spinner screen (it was not positioned at the center)
* Fixed margin top of Complete screen
* Fixed margin top of error container

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/crossDevice/CrossDeviceSubmit/index.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ class CrossDeviceSubmit extends Component {
 
           <div>
             <button
-              className={`${theme.btn} ${theme["btn-primary"]} ${theme["btn-centered"]}`}
+              className={`${theme.btn} ${theme["btn-primary"]} ${theme["btn-centered"]} ${style.btn}`}
```
Comment (rfreitas) [inline src/components/crossDevice/CrossDeviceSubmit/index.js]: not possible to change the style on the theme selectors?
If not than at least we should get rid of some selectors, 4 is quite a bit.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
latest changes breaks buttons on confirmation screen.

**Comment by stefaniacardenas** [comment]
I know :) That's why the title is WIP

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/style.css [hunk 2]
```diff
@@ -65,9 +65,11 @@
...
   border-radius: 2rem;
-  padding: 1.2rem 2rem;
-
+  padding: 0;
   outline: none;
+  @media (--small-viewport) {
+    padding: 1.2rem 2rem;
+  }
 
   &:disabled {
```
[the remark this answers is not in the document]
Comment (stefaniacardenas) [reply src/components/Theme/style.css]: I don't think so

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/style.css [hunk 3]
```diff
@@ -138,6 +136,12 @@
   }
 }
 
+.actions .btn-primary {
```
Comment (rfreitas) [inline src/components/Theme/style.css]: will this work? `.actions` is for the Actions component

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/style.css (hunk 3 above)
Comment (rfreitas) [reply src/components/Theme/style.css]: if this is style is truly just for the Confirm Action button, then it's right to be under the `Confirm/style.css`. The theme style should not have any concept of other components.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/style.css (hunk 3 above)
Comment (stefaniacardenas) [reply src/components/Theme/style.css]: The `actions` class is in the Theme stylesheet to handle cases in which we have more than one button. At the moment it only happens in one case (Confirm screen) but I don't see why we wouldn't reuse this container elsewhere.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://272-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-272-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Theme/style.css (hunk 3 above)
Comment (stefaniacardenas) [reply src/components/Theme/style.css]: I've created a separate issue for that
https://jira.onfido.co.uk/browse/CX-1541

File: src/components/Spinner/style.css [hunk 4]
```diff
@@ -1,12 +1,9 @@
...
 .loader {
   width: 48px;
   height: 48px;
-  margin: auto;
-  position: absolute;
-  top: 0;
-  left: 0;
-  bottom: 0;
-  right: 0;
+  &:extend(.absolute-center);
```
Comment (rfreitas) [inline src/components/Spinner/style.css]: nice!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://272-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**